sophia-fungible-token-contract.md

```diff
@@ -0,0 +1,406 @@
+# TUTORIAL: How to create a Sophia fungible token contract?
+## Tutorial Overview
+This tutorial takes a look at tokens and explains the features and functions of fungible tokens, while provides an understanding of what they are and how developers can work with them.
+## Prerequisites
+- Installed **aecli** (take a look at [this tutorial](https://github.com/aeternity/tutorials/blob/master/account-creation-in-ae-cli.md#installing-aecli) to to remind yourself on installing the javascript version of aecli)
+- Installed **aeproject** (take a look at [this section](https://github.com/aeternity/tutorials/blob/master/smart-contract-deployment-in-aeproject.md#installing-aeproject))
+- Account with testnet/edgenet funds ([How to Get Testnet Funds](https://github.com/aeternity/tutorials/blob/master/get-testnet-tokens.md) - referencing tutorial)
+## Fungible token contract
+Any token contract is a smart contract that contains a map of account addresses and a number called balance. The balance represents a unit of value that is defined by the contract creator. One token contract might use balances to represent physical objects, while another might represent monetary value. Third one might even represent the holder’s reputation. The unit of this balance is commonly called a token.
+Fungible tokens, are such tokens that have one and the same value regardless of which instance of the token is used. AE is a fungible token - meaning that if I give you 1 AE and you give me 1 AE, in essence no value was exchanged.
+Fungible token contracts come with a number of functions to allow users to find out the balances of accounts, as well as to transfer them under varying conditions.
+
+## Building Sophia fungible token contract
+
+### Getting started
+
+The contract starts with the keyword ```contract```, followed by the name of the contract, in this case **FungibleToken**.
+
+Let's define our contract mutable state:
```

Comment:
where to put the code? If it doesn't matter, just mention creating a `.aes` file